profiler/options_test.go

```diff
@@ -62,6 +62,38 @@ func TestOptions(t *testing.T) {
 		assert.Equal(t, expectedURL, cfg.agentURL)
 	})
 
+	t.Run("AgentURL", func(t *testing.T) {
+		os.Setenv("DD_TRACE_AGENT_URL", "https://custom:1234")
+		defer os.Unsetenv("DD_TRACE_AGENT_URL")
```

Comment:
We can use [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) in these tests to make the env var setup a little shorter.